JavaScript/JavaScriptSDK/appInsights.ts

```diff
@@ -190,16 +190,18 @@ module Microsoft.ApplicationInsights {
             if (!Telemetry.PageViewPerformance.isPerformanceTimingSupported()) {
                 // no navigation timing (IE 8, iOS Safari 8.4, Opera Mini 8 - see http://caniuse.com/#feat=nav-timing)
                 _InternalLogging.throwInternalNonUserActionable(LoggingSeverity.CRITICAL,
```

Comment:
My point is - we have telemetry - the number of zeroes. And it is a known problem that doesn't affect many customers. If it go wildly up we can return to discussion what kind of telemetry we need.